std/int128.d

```diff
@@ -263,13 +263,17 @@ public struct Int128
         return this == op2 ? 0 : gt(this.data, op2.data) * 2 - 1;
     }
 
-    /** support signed arithmentic comparison operators < <= > >=
+    /** support arithmentic comparison operators < <= > >=
      * Params: op2 = right hand operand
      * Returns: -1 for less than, 0 for equals, 1 for greater than
      */
-    int opCmp(long op2) const
+    int opCmp(Long)(const Long op2) const
+    if (__traits(isIntegral, Long) && is(Long : long))
```

Comment:
The `&& is(Long : long)` excludes vectors and would exclude cent & ucent if those were implemented.

Minus the exclusion of cent/ucent this could also be written importing `std.traits` as `if (isIntegral!Long || isSomeChar!Long || isBoolean!Long)`.